lib/blog-feed-loader.js

```diff
@@ -0,0 +1,65 @@
+var sanitizeHtml = require('sanitize-html');
```

Comment:
the `contentSnippet` google returns is < 120 characters which is too short for what we need.  I'll switch over to use the stripping code (ie. `myString.replace(/<(?:.|\n)*?>/gm, '');`) instead.